app/components/dashboard/collection_component.html.erb

```diff
@@ -30,7 +25,12 @@
             <% end %>
             <%= render Works::DeleteComponent.new(work: work) %>
           </td>
-          <td><%= render Works::StateDisplayComponent.new(work: work) %></td>
+          <% if work.depositing? %>
+            <td data-controller="state-updates" data-state-updates-work-id="<%= work.id %>" data-state-updates-target="state">
```

Comment:
I wonder if we should rename work-id to object-id so it's more clearly labeled as being used for both works and collections.

Reply:
Ah, nevermind.